### PR TITLE
GPII-3777: Quick Folders (via URL)

### PR DESCRIPTION
**Pull request by Karadaliev**
* Created a new service button called **"Quick Folders"** for access to the quick cloud folder.
* Added all the required changes in order to be able to use **siteConfig** in all of the widgets _(this will be reused in any other button that needs siteConfig)_

### CODE REVIEW

**Comment by Karadaliev** [comment]
All done, the tests will be added shortly. But @amb26, please can start looking into it.

**Comment by amb26** [comment]
ok to test

File: src/main/qss.js [hunk 1]
```diff
@@ -858,9 +858,10 @@ fluid.defaults("gpii.app.qssInWrapper", {
     },
     config: {
         params: {
-            settings: "{qssWrapper}.model.settings"
+            settings: "{qssWrapper}.model.settings",
```
Comment (javihernandez) [inline src/main/qss.js]: extra comma

**Comment by amb26** [comment]
Hi @krisYanachkov , @Karadaliev - this now has conflicts with current master following the merge of GPII-3773, could you please merge up?

**Comment by amb26** [comment]
ok to test

**Comment by amb26** [comment]
Different race condition this time:
````
12:14:44.771:  Data loaded for database 'gpii'...
12:14:49.417:  Test case listener has not responded after 5000ms - at sequence pos 3 of 12 sequence element {
    "event": "{testEnvironment}.events.noUserLoggedIn",
    "listener": "fluid.identity"
} of fixture GPII application integration tests
````

**Comment by amb26** [comment]
ok to test

**Comment by amb26** [comment]
Yet a different race condition - we need to really go over these thoroughly - 
````
13:36:17.658:  FactsManager: Changed interactionsCount: 2
13:36:17.718:  FactsManager: Changed interactionsCount: 3
13:36:17.753:  FactsManager: Changed interactionsCount: 4
13:36:21.422:  contextManager: Updating environment with: {
    "http://registry.gpii.net/common/environment/timestamp": 1552080981422,
    "http://registry.gpii.net/common/environment/timeOfDay": "13:36"
}
13:36:24.928:  Test case listener has not responded after 5000ms - at sequence pos 84 of 174 sequence element {
    "event": "{that}.app.qssWrapper.qssTooltip.events.onDialogShown",
    "listener": "fluid.identity"
} of fixture QSS Widget integration tests
````

It's possible the contextManager is involved, which will be abolished soon

**Comment by amb26** [comment]
ok to test

**Comment by amb26** [comment]
Well that's nice - but let's keep going - ok to test

**Comment by amb26** [comment]
3rd time lucky - ok to test

**Comment by Karadaliev** [comment]
@amb26 seems like everything its alright now, can we proceed with the merge?

**Pull request merged**